pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "tomato-alicat"
+version = "0.1.0"
+description = "Package dedicated to alicat devices for tomato."
+readme = "README.md"
+requires-python = ">=3.8"
+
+authors = [
+    {Kraus Peter , peter.kraus@ceramics.tu-berlin.de,
+    Gbocho Alexandre , gbocho@tu-berlin.de }]
+
+keywords = ["automation", "package", "alicat", "tomato"]
+license = { GNU General Public License v3.0  }
```

Comment:
I am not sure `license` is an allowed keyword in `pyproject.toml`. In any case, you can set it using classifiers, see the bronkhorst example.